### PR TITLE
[Release 1.3.3] Point manifests ref to 1.3.3 tags.

### PR DESCRIPTION
**Pull request by prudhvigodithi**
### Description
https://github.com/opensearch-project/opensearch-build/issues/2101

### Issues Resolved
Part of https://github.com/opensearch-project/opensearch-build/issues/2101

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peterzhuamazon — APPROVED** [review]

File: manifests/1.3.3/opensearch-dashboards-1.3.3.yml [hunk 1]
```diff
@@ -9,35 +9,35 @@ ci:
 components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
-    ref: '1.3'
+    ref: tags/1.3.3
   - name: functionalTestDashboards
     repository: https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
-    ref: '1.3'
+    ref: tags/1.3.3.0
```
Comment (zelinh) [inline manifests/1.3.3/opensearch-dashboards-1.3.3.yml:15]: 1.3.3 tag would be created here.

**Review by zelinh — CHANGES_REQUESTED** [review]

File: manifests/1.3.3/opensearch-dashboards-1.3.3.yml (hunk 1 above)
Comment (peterzhuamazon) [reply manifests/1.3.3/opensearch-dashboards-1.3.3.yml:15]: Change from 1.3.3.0 to 1.3.3 right now.

Comment (prudhvigodithi) [reply manifests/1.3.3/opensearch-dashboards-1.3.3.yml]: Just pushed.

**Review by zelinh — APPROVED** [review]
Thanks.

**Pull request merged**